.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
The [built-in node.js test runner](https://nodejs.org/docs/latest-v18.x/api/test.html) this PR depends on is available starting in the latest LTS version, [v18](https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V18.md#test-runner-module-experimental). This file allows `nvm` to [automatically activate](https://github.com/nvm-sh/nvm#nvmrc) this version for local development.